Zend/zend_execute.c

```diff
@@ -4492,7 +4492,6 @@ static zend_never_inline zend_op_array* ZEND_FASTCALL zend_include_or_eval(zval
 				resolved_path = zend_resolve_path(inc_filename);
 				if (EXPECTED(resolved_path)) {
 					if (zend_hash_exists(&EG(included_files), resolved_path)) {
-						new_op_array = ZEND_FAKE_OP_ARRAY;
```

Comment:
It doesn't look like all paths in `ZEND_INCLUDE_ONCE`/`ZEND_REQUIRE_ONCE` assign `new_op_array` which would previously have been `NULL` but now returns `ZEND_FAKE_OP_ARRAY`. I'm not sure what `ZEND_FAKE_OP_ARRAY` does so I can't say if that's a problem or not.

Reply:
Yes... you are right. I missed those obscure pseudo-`goto` statements (`break`). I dropped this patch. Maybe I'll write a new one to make this piece of code less obscure (and less error-prone).